README.rst

```diff
@@ -70,6 +75,9 @@ For example, in the ``settings.py`` file of your Scrapy project:
         "http": "scrapy_zyte_api.ScrapyZyteAPIDownloadHandler",
         "https": "scrapy_zyte_api.ScrapyZyteAPIDownloadHandler",
     }
+    DOWNLOADER_MIDDLEWARES = {
+        "scrapy_zyte_api.ScrapyZyteAPIDownloaderMiddleware": 1000,
```

Comment:
Not to solve here, by "Scrapy" in the middleware name looks unnecessary. "ScrapyZyteAPI" also may be unnecessary: `scrapy_zyte_api.DownloaderMiddleware`. I think we shouldn't change it in this PR, to maintain consistency with other options.